### PR TITLE
watchexec: add page

### PR DESCRIPTION
**Pull request by WnndGws**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/watchexec.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# watchexec
+
+> Run arbitrary commands when files change.
+> More information: <https://github.com/watchexec/watchexec>.
+
+- Call `ls -la` when any file in the current directory/subdirectories changes:
+
+`watchexec -- {{ls -la}}`
+
+- Run `make` when any JavaScript, CSS and HTML files in the current directory and all subdirectories changes:
```
Comment (zdroid) [inline pages/common/watchexec.md:10]:
```suggestion
- Run `make` when any JavaScript, CSS and HTML files in the current directory:
```

File: pages/common/watchexec.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`watchexec --exts {{js,css,html}} make`
+
+- Run `make` when any file in the `lib` or `src` subdirectories change:
+
+`watchexec --watch {{lib}} --watch {{src}} {{make}}`
+
+- Call/restart `my_server` when any file in the current directory/subdirectories change, sending `SIGKILL` to stop the child process:
```
Comment (zdroid) [inline pages/common/watchexec.md:18]:
```suggestion
- Call/restart `my_server` when any file in the current directory change, sending `SIGKILL` to stop the child process:
```

File: pages/common/watchexec.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# watchexec
+
+> Run arbitrary commands when files change.
+> More information: <https://github.com/watchexec/watchexec>.
+
+- Call `ls -la` when any file in the current directory/subdirectories changes:
```
Comment (zdroid) [inline pages/common/watchexec.md:6]:
```suggestion
- Call `ls -la` when any file in the current directory changes:
```

File: pages/common/watchexec.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# watchexec
+
+> Run arbitrary commands when files change.
+> More information: <https://github.com/watchexec/watchexec>.
+
+- Call `ls -la` when any file in the current directory changes:
+
+`watchexec -- {{ls -la}}`
+
+- Run `make` when any JavaScript, CSS and HTML files in the current directory:
```
Comment (ammgws) [inline pages/common/watchexec.md:10]: Is this missing the word "changes"?

Comment (zdroid) [reply pages/common/watchexec.md:10]: most likely yes

Comment (WnndGws) [reply pages/common/watchexec.md]: You are correct.

**Pull request merged**